lighthouse-core/computed/metrics/responsiveness.js

```diff
@@ -11,50 +11,51 @@
  * user input in the provided trace).
  */
 
+/** @typedef {LH.Trace.CompleteEvent & {name: 'Responsiveness.Renderer.UserInteraction', args: {frame: string, data: {interactionType: 'drag'|'keyboard'|'tapOrClick', maxDuration: number}}}} ResponsivenessEvent */
```

Comment:
nit: did you consider defining this in `.d.ts` (for ergonomic reasons)?

Reply:
> nit: did you consider defining this in `.d.ts` (for ergonomic reasons)?

maybe if it spreads, but so far it's really just needed in this one file and the two files that call it are fine with the implicit type from the function return.

There's also the multi-line jsdoc format, it's just unfortunate that vscode doesn't do better syntax highlighting for it (probably should file that as an issue at some point).